test/config-luatest/config_test.lua

```diff
@@ -391,3 +391,59 @@ g.test_feedback_options = function()
         t.assert_equals(box.cfg.feedback_metrics_limit, 1000000)
     end)
 end
+
+g.test_security_options = function()
+    t.tarantool.skip_if_not_enterprise()
+    local dir = treegen.prepare_directory(g, {}, {})
+    -- guest user is required for luatest.server helper to function properly,
+    -- so it is enabled (`disable_guest: false`) unlike other options.
```

Comment:
We can grant `super` on another user and pass `net_box_credentials = {user = <...>, password = <...>}` to `server:new()` if you like. There is nothing that would actually require `guest` user.

Reply:
It isn't necessary, I wrote the comment to explain why I used default value here, unlike other options.

Reply:
OK, it is up to you. I'm going to resolve the discussion.